packages/orbit-components/src/primitives/TooltipPrimitive/TooltipPrimitive.stories.tsx

```diff
@@ -16,6 +15,11 @@ import { PLACEMENTS } from "../../common/consts";
 
 import TooltipPrimitive from ".";
 
+enum SIZE_OPTIONS {
+  SMALL = "small",
+  MEDIUM = "medium",
+}
+
 const getIcons = defaultIcon => select("Icon", Object.keys(Icons), defaultIcon);
 const getIcon = source => Icons[source];
 
```

Comment:
Well spotted. I noticed that on the Tooltip story the path is correct. However, it only works locally, not in production. So I don't think fixing the name here will solve in production.

I am considering switching to some image from the web, instead of a local file. WDYT?